### PR TITLE
github: fix failing vagrant CI job

### PR DESCRIPTION
**Pull request by gdams**
##### Checklist

- [x] commit message has one of the [standard prefixes](https://github.com/adoptium/infrastructure/blob/master/CONTRIBUTING.md#commit-messages)
- [ ] [faq.md](https://github.com/adoptium/infrastructure/blob/master/FAQ.md) updated if appropriate
- [ ] other documentation is changed or added (if applicable)
- [ ] playbook changes run through [VPC](https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptopenjdk.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)
- [ ] for inventory.yml changes, bastillion/nagios/jenkins updated accordingly

fixes: https://github.com/adoptium/infrastructure/issues/2425

This PR fixes the long-term failing Solaris Vagrant GH actions job. This was broken when we decided to switch to a custom vagrant image that was not available in the environment of the action. This PR switches that logic back to the public image for GH actions CI runs

### CODE REVIEW

**Comment by sxa** [comment]
It does indeed but I've not seen it before.

File: .github/workflows/build_vagrant.yml [hunk 1]
```diff
@@ -22,8 +27,10 @@ jobs:
         brew install --cask virtualbox
 
     - name: Setup Vagrant VM
+      working-directory: ansible
       run: |
-        cd ansible
+        # Temporary fix as the Adopt vagrant vm image is not available on the GitHub Actions executors
```
Comment (zdtsw) [inline .github/workflows/build_vagrant.yml:32]: this probably is not needed after change done for getting solaris box file from /userContent

File: .github/workflows/build_vagrant.yml [hunk 2]
```diff
@@ -8,6 +8,11 @@ on:
     branches:         
     - master
 
+# Cancel existing runs if user makes another push.
+concurrency:
```
Comment (zdtsw) [inline .github/workflows/build_vagrant.yml:12]: good to have this part

**Comment by karianna** [comment]
@gdams Vagrant build is still failing?

**Pull request closed without merging**